Report.md

```diff
@@ -73,6 +73,28 @@ Design Thinking Approach
 ========================
 *3. Describe your approach from a Design Thinking principles perspective. Your Design Thinking approach to address the problem with artefacts*
 
+## **Approach - Design Thinking**
+Design thinking is a non-linear, iterative process which seeks the understanding of users to challenge assumptions, redefine problems and create innovative solutions to prototype and test. This customer-centric approach was undertaken to address the improvements requested by the Major Travel Company for their Call Management Centre (CMC). The design thinking approach undertook consists of the execution of five phases, (1) empathize, (2) define, (3) ideate, (4) prototype and (5) test and by addressing these phrases we derived customer-centric innovation. 
+
+### Empathise
+Phrase one consisted of empathising with stakeholders, this represents anyone who is impacted by the project being developed with a particular focus extended on the end customer. Empathy is a crucial step of achieving a human-centred design process and was utilised to gain insight into users and their needs without introducing any pre imposed bias. When we create solutions, we get better solutions when we think of those who they will affect. To document these perspectives the tool of an empathy map was utilised to collaboratively visualise and share our understanding of stakeholders needs and aid future decision making. This stage allowed us to gain a deeper understanding of the perspective of the users involved and how to shape our solution. 
```

Comment:
Should start with "Phase" along with all the other phases ahaha
Also personally think "the tool of an empathy map" would flow better if it was just "To document these perspectives, empathy maps were utilised..."

Reply:
Right I didn't catch that, definitely, it should be phase - not "phrase". Will make that change. And yep the flow of "the tool of an empathy map" would flow better if it was just "To document these perspectives, empathy maps were utilised.." sounds better - will adjust to that